adr/ADR-32.md

```diff
@@ -108,9 +109,10 @@ All discovery and status responses contain the following fields:
     id: string,
     /**
     * The version of the service
+    * Should be validated using official semver regexp: 
+    * https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
     */
     version: string
```

Comment:
for the version checking - take the string remove any leading and trailing letters
try to parse digit.digit.digit if it works, it is semver.